apps/daffio/src/app/docs/shared/components/guides-viewer/guides-viewer.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { DaffioGuideList } from '../../models/guide-list';
+
+@Component({
+  selector: 'daffio-guides-viewer',
+  templateUrl: './guides-viewer.component.html',
+  styleUrls: ['./guides-viewer.component.scss']
+})
+export class DaffioGuidesViewerComponent {
+
+  @Input() guideList: DaffioGuideList;
+  constructor() { }
```

Comment:
Remove the useless constructor.

Reply:
Addressed in 5de79c050489de6e9ea9890dcbd9152c51a22c04